driver-core/src/main/com/mongodb/internal/connection/DefaultServer.java

```diff
@@ -200,15 +195,6 @@ private boolean isStale(final TopologyVersion currentTopologyVersion, final Topo
         return false;
     }
 
-    public void invalidate(final Throwable t, final int connectionGeneration, final int maxWireVersion,
```

Comment:
Inlined this method, as it was a bit confusing to have another invalidate method

Reply:
Ack